mne/time_frequency/tfr.py

```diff
@@ -192,10 +192,19 @@ def _cwt(X, Ws, mode="same", decim=1, use_fft=True):
         # XXX JRK: full wavelet decomposition needs to be implemented
         raise ValueError('`full` decomposition with convolution is currently' +
                          ' not supported.')
+    if not isinstance(time_range, tuple):
+        raise TypeError('time_range should be a tuple.')
```

Comment:
Duck type - lists or arrays should be fine.

The test below already accomplishes this.

If at all, something like
`all(isinstance(x, int) or x == None for x in time_range)`